extensions/amp-facebook-comments/0.1/amp-facebook-comments.js

```diff
@@ -69,11 +76,39 @@ class AmpFacebookComments extends AMP.BaseElement {
     listenFor(iframe, 'embed-size', data => {
       this./*OK*/changeHeight(data['height']);
     }, /* opt_is3P */true);
+    this.unlistenMessage_ = listen(
+        this.win,
+        'message',
+        this.handleFacebookMessages_.bind(this)
+    );
+    this.toggleLoading(true);
     this.element.appendChild(iframe);
     this.iframe_ = iframe;
     return this.loadPromise(iframe);
   }
 
+  /** @private */
+  handleFacebookMessages_(event) {
+    if (this.iframe_ && event.source != this.iframe_.contentWindow) {
+      return;
+    }
+    if (!getData(event) || !(isObject(getData(event))
```

Comment:
nit: I would cache the value of `getData(event)` in a local variable so you don't have to keep calling `getData`.

Reply:
Done.